<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cell counter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>SRGAP2A_E14.5-E18.5_3-2.png</normaloff>SRGAP2A_E14.5-E18.5_3-2.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max sigma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxSigSpin">
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Min sigma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minSigSpin">
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Overlap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="log_overlap">
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="thresholdSpin">
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>What the right click does:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="rClicktype">
        <item>
         <property name="text">
          <string>Add cell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Remove cell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add 1st box corner</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add 2nd box corner</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add 3rd box corner</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add 4th box corner</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Open image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="imageviewbutton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="13" column="1" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Number of cells per layer</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1" colspan="2">
     <layout class="QGridLayout" name="layout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="table">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>nÂº marked fluorescent cells </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>38</width>
          <height>38</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="nMarkedCells">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>   0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="16">
     <layout class="QVBoxLayout" name="mplvl">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fluorescent marker: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fMarker">
        <item>
         <property name="text">
          <string>RFP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GFP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GFP or RFP</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPushButton" name="FindCells">
      <property name="text">
       <string>Auto find cells</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1" colspan="2">
     <widget class="QPushButton" name="AddClassified">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Save table</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionHow_to_run"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHow_to_run">
   <property name="text">
    <string>how to run</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
